operations/admin.go

```diff
@@ -264,3 +267,94 @@ func updateSettings() cli.Command {
 		},
 	}
 }
+
+func listEvents() cli.Command {
+	return cli.Command{
+		Name:   "list-events",
+		Before: setPlainLogger,
+		Usage:  "print the admin event log",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  joinFlagNames(startTimeFlagName, "t"),
```

Comment:
flag names should be function constants, or defined in the flags file?

Reply:
this one is defined in the flags file

Reply:
does it make sense to define the flag itself there too?